src/SreBot.Web/Controllers/NewRelicController.cs

```diff
@@ -17,10 +17,20 @@ public NewRelicController(IManageNewRelic newRelicService)
 		}
 
 		[Route("")]
-		public async Task<IActionResult> Index()
+		public async Task<IActionResult> Index(string selectedAccount = null)
 		{
-			var response = await this.newRelicService.GetUnhealthyApplications();
-			return this.View(response.Applications);
+			var accounts = newRelicService.GetAccountNames();
```

Comment:
I don't think we're accessing the response of this object? Looks like might of been left behind after a refactor...

Reply:
Yeah, good catch. I'll remove this line